src/main/webapp/i18n/de/imprint.json

```diff
@@ -0,0 +1,13 @@
+{
+    "artemisApp": {
+        "imprint": {
+            "title": "Impressum",
+            "updateImprint": "Impressum aktualisieren",
+            "unsavedChangesWarning": "Das Impressum enthält ungespeicherte Änderungen. Möchtest du wirklich die Sprache wechseln und alle ungespeicherten Änderungen verwerfen?",
+            "language": {
+                "de": "Deutsch",
+                "en": "Englisch"
+            }
```

Comment:
Why is the `language` key defined again here? Can’t the one from `legal.json` be reused?

Same in the English translation files.

---

Would it make sense to merge `{imprint,legal,privacyStatement}.json` all into `legal.json`, since they are quite small and are probably not expected to receive significant additions in the future?

Reply:
I also thought about it, gonna change it then.